test/sim/items/ejectpack.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const assert = require('./../../assert');
+const common = require('./../../common');
+
+let battle;
+
+describe("Eject Pack", function () {
+	afterEach(function () {
+		battle.destroy();
+	});
+
+	it("should switch out the holder when its stats are lowered", function () {
+		battle = common.createBattle();
+		battle.setPlayer('p1', {team: [
+			{species: 'Machop', ability: 'noguard', moves: ['leer']},
+		]});
+		battle.setPlayer('p2', {team: [
+			{species: 'Magikarp', ability: 'swiftswim', item: 'ejectpack', moves: ['splash']},
+		]});
+		battle.makeChoices();
+		assert.strictEqual(battle.p2.requestState, 'switch');
```

Comment:
Travis errored here: "AssertionError [ERR_ASSERTION]: Input A expected to strictly equal input B:
+ expected - actual
- 'move'
+ 'switch'"

Reply:
This happens because there's only one Pokémon on Magikarp's team; adding another Pokémon (so Magikarp can switch, so Eject Pack can activate) makes this test pass.